src/main/java/seedu/address/model/expense/RecurringExpenseManager.java

```diff
@@ -209,31 +197,24 @@ public boolean equals(Object object) {
         if (!recurringExpense.startDate.isEqual(this.startDate)) {
             return false;
         }
-
         if (recurringExpense.amount != this.amount) {
             return false;
         }
-
         if (!recurringExpense.nextExpenseDate.isEqual(this.nextExpenseDate)) {
             return false;
         }
-
         if (recurringExpense.recurringExpenseType != this.recurringExpenseType) {
             return false;
         }
-
         if (!recurringExpense.expenseCategory.equals(this.expenseCategory)) {
             return false;
         }
-
         if (!recurringExpense.expenseName.equals(this.expenseName)) {
             return false;
         }
```

Comment:
There could be a less repetitive way of doing this, can consider refactoring, but it is not a big issue.

Reply:
i agree, i'll refactor it